ducktape/services/service.py

```diff
@@ -25,18 +25,24 @@
 # that there won't be resource conflicts: the cluster tests are being run on
 # should be large enough to use one instance per service instance.
 
-from ducktape.logger import Logger
 
-
-class Service(Logger):
-    def __init__(self, cluster, num_nodes):
-        self.num_nodes = num_nodes
-        self.cluster = cluster
+class Service(object):
+    def __init__(self, service_context):
+        """
+        :type service_context: ServiceContext
+        """
+        self.num_nodes = service_context.num_nodes
+        self.cluster = service_context.cluster
+        self.logger = service_context.logger
 
     def start(self):
         """Start the service running in the background."""
         self.nodes = self.cluster.request(self.num_nodes)
         for idx, node in enumerate(self.nodes, 1):
+
+            # Remote accounts utilities should log where this service logs
+            node.account.set_logger(self.logger)
```

Comment:
Interestingly this immediately caught the fact that no Service subclass was calling super(...).stop() in its own stop method